src/wallet.cpp

```diff
@@ -3702,7 +3702,9 @@ void CWallet::AutoCombineDust()
 
 bool CWallet::MultiSend()
 {
-    if (IsInitialBlockDownload() || IsLocked()) {
+    // Stop the old blocks from sending multisends
+    if (chainActive.Tip()->nTime < (GetAdjustedTime() - 300) || IsLocked()) {
+        LogPrintf("Multisend: disabled on tx's that are being read on initial block loading, %d", chainActive.Tip()->nTime);
```

Comment:
This could get pretty spammy if resyncing the chain while having multisend activated. I think we should either remove this print statement or change it to something like `LogPrint('debug', "Multisend: .....`

Reply:
I agree. I am not sure this there is merit to print it with the debug flag. I was using this to verify that multi-send still worked. I think removing it completely is the best option.